README.md

```diff
@@ -1,18 +1,30 @@
-# AsyncApi Simulator
-___
+<h5 align="center">
+  <br>
+  <a href="https://www.asyncapi.org"><img src="https://github.com/asyncapi/parser-nodejs/raw/master/assets/logo.png" alt="AsyncAPI logo" width="200"></a>
+  <br>
+  Simulator
+</h5>
+<p align="center">
+  <em>The official simulator for simulating scenarios</em>
```

Comment:
simulator for simulating 😄 I have no suggestion but we definitely need something better 😄

Reply:
`... running scenarios`? 🤔

Reply:
it doesn't have to be super short, you can expand to 2-3 sentences

Reply:
```suggestion
  <em>The official simulator for AsyncAPI. It can be used to run different well-defined scenarios, such as user-flows, edge cases, error handling, and other odd-cases that require specific execution of steps against your services.</em>
```

Maybe something along the lines of this? cc @derberg @NektariosFifes

Reply:
Lets optimize the documentation later as the tool progresses